packages/operator/pkg/controller/adapters/v1/deployment/model_deployment.go

```diff
@@ -0,0 +1,233 @@
+package deployment
+
+import (
+	"context"
+	hashutil "github.com/odahu/odahu-flow/packages/operator/pkg/utils/hash"
+	odahuv1alpha1 "github.com/odahu/odahu-flow/packages/operator/api/v1alpha1"
+	deployment_types "github.com/odahu/odahu-flow/packages/operator/pkg/apis/deployment"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/controller/types"
+	odahu_errors "github.com/odahu/odahu-flow/packages/operator/pkg/errors"
+	kube_client "github.com/odahu/odahu-flow/packages/operator/pkg/kubeclient/deploymentclient"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/service/deployment"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"time"
+)
+
+var (
+	log = logf.Log.WithName("model-deployment-adapter")
+)
+
+type KubeEntity struct {
+	obj        *deployment_types.ModelDeployment
+	service       deployment.Service
```

Comment:
In future we replace service on Http client to avoid direct calls to DB. So the idea is a bit different. Api Server provide API to push status and retrieve entities, our HTTP API is written over service layer, not DB

Reply:
OK, got that :+1: